jsonschema/compat.py

```diff
@@ -40,6 +43,9 @@ def urlsplit(url):
     return SplitResult(scheme, netloc, path, query, fragment)
 
 
+DefragResult = namedtuple('DefragResult', 'url fragment')
```

Comment:
Previously `urldefrag` was being used in a few places, I believe it's only being used in one place now, so this is less necessary (and I believe it is only for attribute access). I can remove it (it was actually pulled in from the cherry-pick of the previous changes from ankostis).

I must say I'm surprised by this response. I consider namedtuple one of the nice parts of python, and one of the most underused.  Replacing this with a class would be pretty verbose and lots of boilerplate (slots=(...), assignment in the construction, equality and comparison functions). In my experience keeping data objects separate from functionality that operations on them also makes code a lot more reasonable (which is why I find `namedtuple` to be so valuable, it lets you create data objects very easily).

What's wrong with namedtuple?